wiki/Sitemap/en.md

```diff
@@ -330,6 +330,8 @@ tags:
     -   [osu! Riverside](/wiki/Tournaments/osu!_Riverside)
         -   [MALE](/wiki/Tournaments/osu!_Riverside/MALE)
         -   [SWEAT](/wiki/Tournaments/osu!_Riverside/SWEAT)
+    -   [osu! UCI](/wiki/Tournaments/osu!_UCI)
+        -   [iDOL@NSTER](/wiki/Tournaments/osu!_UCI/iDOL@NSTER)
```

Comment:
The ASC says to not use any percent encoding characters, that `@` is going to show up as `%40`. Did we want to ignore it and change it in the ASC later?

Reply:
exclamation marks are already allowed so I don't really see the point of this

Reply:
Oh, I thought exclamation marks wasn't encoded, otherwise it wouldn't have been listed there. Alright, will make some changes to that section later.